reporting.md

```diff
@@ -499,11 +497,11 @@ Figure: JSON Report Format
     Presently, the only three valid choices are `tlsa`, `sts`, and the literal
     string `no-policy-found`. It is provided as a string.
 * `policy-string`: The string serialization of the policy, whether TLSA record
-    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with
-    [SP]. TODO: Help with specifics.
-* `domain`: The Policy Domain upon which the policy was applied. For messages
-    sent to `user@example.com` this field would contain `example.com`. It is
-    provided as a string.
+    ([@!RFC6698] section 2.3) or MTA-STS policy. Any linefeeds from the original
+    policy MUST be replaced with space.
```

Comment:
Sure. See the string representation section here: https://tools.ietf.org/html/rfc7159#section-7. 

My point was that in JSON we don't have to replace the line feed character (since that means mangling the original policy); we merely need to escape it (e.g. "/n" and "/r"). I am not sure we need to specify this very carefully since "string representation of the policy" in "JSON" more or less implies as much, no?